### PR TITLE
Robust

### PR DESCRIPTION
**Pull request by beyang**

### CODE REVIEW

**Comment by MaikuMori** [comment]
"Duplicate" `virtualenv` was there so that the dependencies don't conflict. Now if the source unit has dependency on another `jedi` version it will break the toolchain. Also analyzing 2 python projects with different library versions now is slower. Upside is you save some space.

File: grapher/file_grapher.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 from collections import namedtuple
 import os
 import re
+import sys
 
 import jedi
 
-
```
Comment (MaikuMori) [inline grapher/file_grapher.py]: This is as per [`PEP8`](https://www.python.org/dev/peps/pep-0008/#blank-lines) style guidelines. Which is the "official" Python style guide.

> Surround top-level function and class definitions with two blank lines.

**Comment by MaikuMori** [comment]
Also let me know if you need some assistance with this project. My memory has faded a bit, but maybe there is something I can help with.

**Comment by beyang** [comment]
Thanks for taking a look, Miks. Will let you know if there's anything where your help would be appreciated :)

The duplicate virtualenv is no longer as necessary since we are targeting a docker-based environment.

**Pull request merged**